docs/tutorials/Raster-data-Lambert93.md

```diff
@@ -161,7 +164,7 @@ The camera has been zoomed in to get a more precise rendering of the elevation v
 
 ## Result
 
-With this tutorial, we learnt how to use iTowns `ColorLayer` and an `ElevationLayer` to display data in a local conic projection. 
+With this tutorial, we have learn how to use iTowns `ColorLayer` and an `ElevationLayer` to display data in a local conic projection. 
```

Comment:
`have learnt`, if I recall my irregular verbs, the past participle takes a `t` :slightly_smiling_face:

Reply:
yep you're right, my former english teachers would not be proud :grin: